package com.zccoder.javabase.day10;
/*
第一个格式：
*/
//try
//{
//    
//}
//catch ()
//{
//}
//第二个格式：
//try
//{
//    
//}
//catch ()
//{
//}
//finally
//{
//}
//第三个格式：
//try
//{
//    
//}
//finally
//{
//}
//记住一点：catch是用于处理异常。如果没有catch就代表没有被处理过，如果该异常时检测时的异常，就必须声明出去。
//class Demo
//{
//    public void method()
//    {
//        try
//        {
//            throw new Exception();
//        }
//        finally
//        {
//            //关资源。
//        }
//        
//    }
//}



public class ExceptionDemo6
{
    public static void main(String[] args) 
    {
        System.out.println("Hello World!");
    }
}
